ci.py

```diff
@@ -232,6 +232,8 @@ def check_deps(self):
             else:
                 target_url = next(item for item in self.config['deps'] if item["name"] == dep['name'])['url']
             repo_path = '/'.join(target_url.split('/')[-2:])
+            if '.git' in repo_path:
```

Comment:
Without this, the build would fails with a 404 not being able to find the repository in case it's name ends with '.git'.